composer.json

```diff
@@ -6,7 +6,6 @@
     "license": "GPL-2.0-or-later",
     "minimum-stability": "dev",
     "require": {
-        "cweagans/composer-patches": "^1.6",
```

Comment:
So this line isn't needed. 

If it's been included an enabled in a parent composer.json it will work (see caveat about install order in comment coming).
If the parent composer.json doesn't have it, requiring it doesn't make sense.